lib/gui/pages/settings/controllers/settings.js

```diff
@@ -53,72 +54,50 @@ module.exports = function(WarningModalService, SettingsModel, AnalyticsService)
   this.model = SettingsModel;
 
   /**
-   * @summary Enable a dangerous setting
+   * @summary Toggle setting
    * @function
    * @public
    *
-   * @param {String} name - setting name
-   * @param {Object} options - options
-   * @param {String} options.description - modal description
-   * @param {String} options.confirmationLabel - modal confirmation label
+   * @description
+   * If warningOptions is given, it should be an object having `description` and `confirmationLabel`;
+   * these will be used to present a user confirmation modal before enabling the setting.
```

Comment:
Maybe this could also mention that no confirmation modal is shown when disabling a 'dangerous' setting?